code/modules/antagonists/pirate/pirate_shuttle_equipment.dm

```diff
@@ -126,6 +126,12 @@
 	var/cooldown = 300
 	var/next_use = 0
 
+/// Surgery disk for the space IRS (I don't know where to dump them anywhere else)
+/obj/item/disk/surgery/irs
+	name = "Advanced Surgery Disk"
+	desc = "A disk that contains advanced surgery procedures, must be loaded into an Operating Console."
+	surgeries = list(/datum/surgery/advanced/lobotomy, /datum/surgery/advanced/bioware/vein_threading, /datum/surgery/advanced/bioware/nerve_splicing, /datum/surgery_step/heal/combo/upgraded, /datum/surgery_step/pacify, /datum/surgery_step/revive)
```

Comment:
Why do they have surgeries?

Reply:
For more durability, if one of them dies you can revive them with the medical equipment onboard relatively easily.